Create example beacon log in git rather than dropbox

teesloane opened the pull request:
@benjmarshall raised a good point: we are not including a beacon_log in the project directory; at the early stage of the project we were using dropbox to auto sync files. I think we should just use git for now to keep things simple, and think about how a `beacon_log.json` file might be stored / synced between projects in the future.

teesloane committed:
Create example beacon log in git rather than dropbox

jbydeley commented on main.go:
```diff
@@ -49,7 +47,7 @@ type Log struct {
 // ============================
 
-var logFilePath = path.Join(os.Getenv("HOME"), "/Dropbox/The Heap/Beacon/beacon_log.json")
+// var logFilePath = path.Join(os.Getenv("HOME"), "/Dropbox/The Heap/Beacon/beacon_log.json")
```
Do you want to keep commented out code or should this just be removed?

teesloane replied:
Ah good catch, thanks for reviewing @jbydeley ! Will fix now! (and add you as a contributor on our readme for reviewing  PRs :)

teesloane committed:
remove old dropbox file link commented out code (+2 squashed commits)
Squashed commits:
[f435eb5] Add @benjmarshall as a contributor
[7ae568f] Add @jbydeley as a contributor

teesloane commented:
Thanks for reviewing @jbydeley I think it's time to 🚢 🚢 🚢
